bin/a0-ext

```diff
@@ -52,5 +54,31 @@ prog
             });
           });
       });
+  })
+  .command('build:client', 'Compile your React Frontend into an Auth0 Extension Client')
```

Comment:
Will all extensions need to be updated with a `build:client` command? If so I'd like to make sure it fits into @shawnmclean's plans for CI/CD

Reply:
Yes, but it's just `"client:build": "a0-ext build:client ./client/app.jsx ./dist",` vs old `"client:build": "cross-env NODE_ENV=production webpack --config ./build/webpack/config.prod.js --colors -p",`